nixos/tests/ly.nix

```diff
@@ -0,0 +1,39 @@
+{ system ? builtins.currentSystem,
```

Comment:
Please add this test to `ly`'s `passthru.tests` so a bump from r-ryantm also runs this test.

Reply:
I find that neither sddm and lightdm set `passthru.tests`, and the tests is for nixos, I don't think it should be added for the package.

Reply:
But that's the point of `passthru.tests` basically.

NixOS in this context works like a framework.